java/sage/Person.java

```diff
@@ -196,6 +235,14 @@ public boolean isPersonIdentical(Person testMe)
       for (int i = 0; i < awardNames.length; i++)
         if (awardNames[i] != testMe.awardNames[i])
           return false;
+      for (int i = 0; i < headshotUrls.length; i++)
+      {
+        if (headshotUrls[i].length != testMe.headshotUrls[i].length)
+          return false;
+        for (int j = 0; j < headshotUrls[i].length; j++)
+          if (headshotUrls[i][j] != testMe.headshotUrls[i][j])
```

Comment:
My bad....I didn't see the other for loop in there. This is fine (I thought you were comparing the two arrays with ==...but that is not what you are doing).